cmake/edge_configure.cmake

```diff
@@ -152,6 +152,7 @@ if (${FIRMWARE_UPDATE})
   if (NOT FOTA_ENABLE)
      MESSAGE("Update client hub selected.")
      add_definitions ("-DMBED_CLOUD_CLIENT_SUPPORT_UPDATE=1")
+     add_definitions("-DPAL_DNS_API_VERSION=1")
```

Comment:
V2 gives the better IPv6 support, right? We should take that into use.

Reply:
Why not to use the v3 which is already the default in 4.11 ?

Reply:
Getting this error. We tried on version 2 first but it's not supported on Linux. So V2 or greater is only supported on Mbed.
 #error "Async PAL DNS API v2 or greater is only supported on Mbed."

Reply:
That's weird. Please investigate as according to 4.11 release notes:
`[Linux] Enabled PAL_DNS_API_VERSION 3 by default for Linux configurations.`

Reply:
@teetak01 - yep, does not sound right at all. But, async api sounds like something else, any ideas?

Reply:
Either use the Legacy sync API v=0, or use the improved v3.

Reply:
changing into 3 produces the same result
#error "Async PAL DNS API v2 or greater is only supported on Mbed."
 #error "Async PAL DNS API v2 or greater is only supported on Mbed."

Reply:
Remove the dependencies to Uc-hub and it will work.

Reply:
I am not sure if we can do that as of now. We will be depreciating UC-HUB progressively but not in this build.

Reply:
Which configs would use anything but FOTA_FW now?  LmP is already fully FOTA_FW.

Reply:
This flag is just the case of UC_HUB only. In FOTA_FW, we are using the default PAL_DNS_API_VERSION. So we are using v1 in the case of UC_HUB and and the default version of MCC in FOTA_FW.